### ISSUE TITLE
Add a language section above first-timers only issue

### ISSUE THREAD

**Issue by govindgoel** [publiclab/community-toolbox#337]
### Please describe the idea:
It would be great if we can add a section that fetches what are the languages used in that repository.It would be great as it will give a idea to newcomers on which project they should contribute  ,and   `github-api-simple` has api for this purpose also
Something like this
![enhance](https://user-images.githubusercontent.com/52847415/74136764-adf82880-4c14-11ea-8e21-3a0cea87757a.png)

### Please show us where to look

https://code.publiclab.org/ ...

### What's your PublicLab.org username?

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by akanksha-v** [comment]
@govindgoel May I work upon this?
And also, do we need to show languages for every repo of publiclab or just community toolbox?

**Comment by govindgoel** [comment]
@akanksha-v  Yeah you can work on this, it would be better if we go for languages of each repo if you have some other sort of design from which is discussed do share.

**Comment by YogeshSharma01** [comment]
Hi, @akanksha-v You working on this?

**Comment by akanksha-v** [comment]
@YogeshSharma01 you can work upon this

**Comment by YogeshSharma01** [comment]
@akanksha-v Ok cool!

**Comment by YogeshSharma01** [comment]
hi, @govindgoel we can fetch the name of the repo, the link, and the language for all the repository in the public lab by using GitHub API.
We can use that data to display the hyperlinked name and the corresponding language for that repo. All this data in this section can be displayed just like the FTO issues one right below it. The user will be able to go to that repo directly by clicking on the hyperlinked name of the repo.

![OK](https://user-images.githubusercontent.com/51679200/103934592-4069ba80-514b-11eb-9e76-24a60877832f.png)

**Comment by YogeshSharma01** [comment]
@govindgoel, @akanksha-v what do you think about it?

**Comment by govindgoel** [comment]
@YogeshSharma01 yeah it can be done, I was thinking of an approach where language can be specified somewhere else as it may look crowded at this place.

**Comment by YogeshSharma01** [comment]
hi, @govindgoel no I don't think it would be crowded like the repo name with the link of that particular repo will be set at the left corner in the language section and the language is used in that particular repo will be placed at the right corner and we can also use some special icons to represent the language used in the repo. if you have thought of something else, please do recommend it.

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel what do you think? Do we need to add anything more?

https://user-images.githubusercontent.com/51679200/104765656-3cfbb200-578f-11eb-935f-12c795de4e15.mov

**Comment by YogeshSharma01** [comment]
![Language-Image](https://user-images.githubusercontent.com/51679200/104766998-77664e80-5791-11eb-920e-602cc0a00ecd.png)

**Comment by YogeshSharma01** [comment]
I give a pull request, please review @govindgoel 
link to pull request #422

**Comment by govindgoel** [comment]
It seems crowded and will lead to more scrolling, it would be great if we
can find a way to show the language with respect to the repo name in the
dropdown.

Warm Regards,
Govind Goel

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel 

https://user-images.githubusercontent.com/51679200/104820304-e27b5800-5859-11eb-9f3f-c3ef6c39221b.mov


What do you think is this fine? Now we don't need to scroll the Language-section and also it don't look crowedy now.

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel please review the changes I have updated the changes. 
link to pull request https://github.com/publiclab/community-toolbox/pull/422

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel  please review 
link to pull requests https://github.com/publiclab/community-toolbox/pull/422, https://github.com/publiclab/community-toolbox/pull/420

**Comment by jywarren** [comment]
Hi, I'm sorry I missed this conversation! I really appreciate it. However, i wonder if we need to put this section at the bottom of the page and link down to it from the new navbar. Because we really want people to be able to see the first-timers-only issues as close to the top as possible! 

Another possibility is that in the first-timers-only issues explanation text, as shown below the header in this screenshot, we could add the line "You can also _sort by language_" which then links down to the languages section. How does that sound? 

![image](https://user-images.githubusercontent.com/24359/106313893-ef735080-6236-11eb-87b7-511ff2944d20.png)

**Comment by YogeshSharma01** [comment]
Hi, @jywarren that sounds awesome ok I'll place this section at the bottom of the page and add the line "You can also sort by language" which then links down to the languages section.
Thanks
warm regards 
Yogesh Sharma

**Comment by YogeshSharma01** [comment]
@jywarren review the changes ...

https://user-images.githubusercontent.com/51679200/106316610-08a9ea80-6293-11eb-8f1c-673fd8edcd37.mov

**Comment by jywarren** [comment]
This looks great! One more request though -- can we add the text description "Hover over each project to see what languages it uses"? I might prefer a visible label though because this won't work in mobile, you know? But, tell me what you think! Could we add labels in this style, maybe?

![image](https://user-images.githubusercontent.com/24359/106339997-4f352000-6266-11eb-96fc-7de2286bfc3f.png)

I'm just worried about the visibility on mobile, you know? Thank you!!

**Comment by YogeshSharma01** [comment]
Hi @jywarren, Yes you're right it won't work on mobile we can add labels in this style like repo name at left corner and the language used at right side. 
![Lang-label](https://user-images.githubusercontent.com/51679200/106362468-f8ecdd80-6348-11eb-8ff3-12ec28dc281e.gif)
what do you think ?

**Comment by govindgoel** [comment]
@jywarren  I have another idea to implement this, we can think of displaying it in the dropdown list of repo's itself then there will be no need for a new section.

**Comment by YogeshSharma01** [comment]
@govindgoel like we can do that but I think it would look crowded like all the repo names within a single dropdown? can use please show some mockup or sketch something like that brother?

**Comment by jywarren** [comment]
Hi @YogeshSharma01 i like your last design, let's give it a try! @govindgoel i appreciate your idea as well; one way we could do this is to consider having this selection show only the top 3-5, and have a button that says "See all" or "Expand to see all" which opens the full box. What do you think?

**Comment by YogeshSharma01** [comment]
Hi, @jywarren and @govindgoel sorry for the late reply actually I was busy with my exams Please review this. 

https://user-images.githubusercontent.com/51679200/108588095-66ef5800-737d-11eb-8c91-3d629e8f1417.mov
